azure/v11.4.0/release.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v11.4.0
+spec:
+  state: active
+  date: 2020-06-23T15:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: chart-operator
+    version: 0.13.0
+  - name: coredns
+    componentVersion: 1.6.5
+    version: 1.1.9
```

```suggestion
    version: 1.1.10
```
Upgrade to CoreDNS App 1.1.10